features/cli/build.feature

```diff
@@ -474,26 +474,27 @@ Feature: build 'apps' with CLI
       | resource_name | ruby-hello-world                         |
       | p             | {"spec": {"runPolicy": "Parallel"}} |
     Then the step should succeed
-    Given I run the steps 3 times:
+    Given I run the steps 2 times:
```

Comment:
@xiuwang LG, just curious why changed the iterations from 3 to 2?

Reply:
2 builds are enough once time